api/data_explorer/util/elasticsearch_util.py

```diff
@@ -267,6 +267,9 @@ def get_time_series_vals(es, field_name, mapping):
     time_series_vals = submapping.keys()
     assert '_is_time_series' in time_series_vals
     time_series_vals.remove('_is_time_series')
+    has_unknown = 'unknown' in time_series_vals
```

Comment:
Is 270-272 needed? Since now it's `Unknown`...

Reply:
In order to sort the times, I think it's still easier to remove Unknown and then convert everything to int/float?

Reply:
SG. Ignore my comment -- I mistakenly thought it was `Unknown` in ES index